ccf_specs.md

```diff
@@ -515,6 +515,16 @@ Cadence data is encoded depending on its type.
 For example, Cadence `UInt8` is encoded as CBOR positive integer, Cadence `String` is encoded as CBOR text string,
 Cadence `Address` is encoded as CBOR byte string, and Cadence struct data is encoded as an array of its raw field data.
 
+### Cadence Type and TypeValue
+
+Cadence types and Cadence `TypeValue` are encoded differently.  They contain different data because they are used differently.
```

Comment:
```suggestion
Cadence types and Cadence type values (run-time types) are encoded differently.  They contain different data because they are used differently.
```

Reply:
@turbolent 

> Cadence types and Cadence type values (run-time types) are encoded differently.

Can Cadence type values be static types as well as run-time types?

Reply:
Cadence type values are also called "run-time types", and they are static types reified as a value